presto-hive/src/main/java/com/facebook/presto/hive/SyncPartitionMetadataProcedure.java

```diff
@@ -28,6 +28,7 @@
 import com.facebook.presto.spi.procedure.Procedure.Argument;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Lists;
```

Comment:
Commit title too long. Upper case the first Letter.

Reply:
fixed